scipy/stats/_stats_mstats_common.py

```diff
@@ -44,17 +44,27 @@ def linregress(x, y=None):
 
     Examples
     --------
+    >>> from numpy import random
     >>> from scipy import stats
-    >>> np.random.seed(12345678)
-    >>> x = np.random.random(10)
-    >>> y = np.random.random(10)
-    >>> slope, intercept, r_value, p_value, std_err = stats.linregress(x,y)
+    >>> random.seed(12345678)
+    >>> x = random.random(10)
+    >>> y = random.random(10)
+    >>> slope, intercept, r_value, p_value, std_err = stats.linregress(x, y)
 
-    # To get coefficient of determination (r_squared)
+    To get coefficient of determination (r_squared)
 
     >>> print("r-squared:", r_value**2)
     ('r-squared:', 0.080402268539028335)
 
+    Plot the data along with the fitted line
+
+    >>> from matplotlib import pyplot as plt
+    >>> plt.plot(x, y, 'o', label='Original data', markersize=10)
+    >>> plt.plot(x, intercept + slope*x, 'r', label='Fitted line')
+    >>> plt.xlim(-1, 4)
+    >>> plt.ylim(-2, 5)
```

Comment:
the plot looks better this way. but i  guess to save some lines of code i'll remove them. the default range is good enough for this example.